website/client/src/components/static/home.vue

```diff
@@ -844,6 +860,14 @@ export default {
       if (this.username.length < 1) return false;
       return !this.usernameValid;
     },
+    passwordValid () {
+      if (this.password.length <= 0) return false;
+      return this.password.length >= MINIMUM_PASSWORD_LENGTH;
+    },
+    passwordInvalid () {
+      if (this.password.length <= 0) return false;
```

Comment:
Is this <=0 line correct? Doesn't it mean that `passwordInvalid` will be `false` (i.e., password will be considered valid) if the length is 0 or less?

Are this line and also line 864 needed at all? I think the checks against MINIMUM_PASSWORD_LENGTH will handle all the cases.

Reply:
> Is this <=0 line correct? Doesn't it mean that passwordInvalid will be false (i.e., password will be considered valid) if the length is 0 or less?

The idea here is (taken from other fields) not to show the error class until the user has typed something int to field